### PR TITLE
Add DeepL

### PR DESCRIPTION
**Pull request by mtrucc**
**Issue:** closes #

**Similarweb rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3341,6 +3341,11 @@
             "hex": "007CFF",
             "source": "https://commons.wikimedia.org/wiki/File:Deepin_logo.svg"
         },
+        {
+            "title": "DeepL",
+            "hex": "002746",
```
Comment (adamrusted) [inline _data/simple-icons.json:3346]:
```suggestion
            "hex": "0F2B46",
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -3341,6 +3341,11 @@
...
+            "source": "https://upload.wikimedia.org/wikipedia/commons/e/ed/DeepL_logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:3347]:
```suggestion
            "source": "https://www.deepl.com"
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the contribution @mtrucc!
There's a couple of changes I'd make to the JSON - and that in turn would impact the icon itself.
Would you mind updating the JSON, and re-generating from the new source? 🙏

File: _data/simple-icons.json (hunk 2 above)
Comment (mtrucc) [reply _data/simple-icons.json:3347]: Thank you for further improvement

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
They have a [press page](https://www.deepl.com/press.html) so we should probably use what's available there, and update the source accordingly.

Using the  "[Logo DeepL (svg)](https://www.deepl.com/img/press/logo_DeepL.svg?v=1)" file (which the [Wikimedia article](https://commons.wikimedia.org/wiki/File:DeepL_logo.svg) claims to be sourced from) for comparison, there are [significant differences in your version](https://github.com/simple-icons/simple-icons/assets/15157491/37bbc038-14be-410a-bbc1-57f10de8a707) (in red), @mtrucc - would you mind taking another pass at this, please?

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @mtrucc?

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks for updating, @mtrucc. However the maximum precision we allow for an icon is 5 decimal places.

File: _data/simple-icons.json [hunk 3]
```diff
@@ -3571,6 +3571,11 @@
             "hex": "007CFF",
             "source": "https://commons.wikimedia.org/wiki/File:Deepin_logo.svg"
         },
+        {
+            "title": "DeepL",
+            "hex": "002746",
+            "source": "https://www.deepl.com"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:3577]:
```suggestion
            "hex": "0F2B46",
            "source": "https://www.deepl.com/press.html"
```

**Review by PeterShaggyNoble — APPROVED** [review]
Icon looks good to me now, @mtrucc 👍🏻 And I agree with your choice to use the full logo as they don't make the icon itself available on their press page. Just need to update the `hex` & `source` and this is ready to merge.

**Pull request merged**